packages/account-v2/src/App.tsx

```diff
@@ -5,9 +5,11 @@ import './index.scss';
 
 const App: React.FC = () => (
     <APIProvider standalone>
-        <div className=' text-solid-slate-500 text-heading-h1'>Account V2</div>
-        <div className='p-300'>
-            <BrandDerivLogoCoralIcon height='120px' width='120px' />
+        <div className='quill-design'>
```

Comment:
All quill tokens will be scoped in apps that is wrapped under `quill-design` class name
<img width="441" alt="Screenshot 2024-01-04 at 4 21 12 PM" src="https://github.com/binary-com/deriv-app/assets/103016120/1779719e-0a4b-447d-b3c6-59b7c8ea885f">

Reply:
If you need to use quill design tokens, your app must be wrapped with a classname of `quill-design`

Reply:
I think its better to name it the same, so others don't have to come up with a new name each time they wanna scope the quill tokens 😅